compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/ProjectDirConstants.java

```diff
@@ -98,4 +98,6 @@ private ProjectDirConstants() {
 
     public static final String BLANG_PKG_DEFAULT_VERSION = "0.0.0";
 
+    public static final String MODULES_ROOT = "modules";
```

Comment:
Shall we move this constant inside the project-api module. Think this effort is as a clean up as well.

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-lang/pull/25337